docs/reference/workflows/monobehaviour/worker-connectors.md

```diff
@@ -0,0 +1,140 @@
+        connParams.Network.ConnectionType = NetworkConnectionType.Kcp;
+
+        // Create the builder and set the connection parameters
+        var builder = new SpatialOSConnectionHandlerBuilder()
+            .SetConnectionParameters(connParams);
+
+        if (!Application.isEditor)
+        {
+            // If we aren't in the Editor, I want to configure my connection via the command line.
+            var initializer = new CommandLineConnectionFlowInitializer();
+
+            // There are different connection flows for different scenarios.
+            switch (initializer.GetConnectionService())
+            {
+                case ConnectionService.Receptionist:
+                    builder.SetConnectionFlow(new ReceptionistFlow(CreateNewWorkerId(WorkerType), initializer));
+                    break;
+                case ConnectionService.Locator:
+                    builder.SetConnectionFlow(new LocatorFlow(initializer));
+                    break;
+                case ConnectionService.AlphaLocator:
+                    builder.SetConnectionFlow(new AlphaLocatorFlow(initializer));
+                    break;
+                default:
+                    throw new ArgumentOutOfRangeException();
+            }
+        }
+        else
+        {
+            // If we are in the Editor, simply connect via the Receptionist.
+            builder.SetConnectionFlow(new ReceptionistFlow(CreateNewWorkerId(WorkerType)));
+        }
+
+        // Asynchronously connect to the Runtime.
+        await Connect(builder, new ForwardingDispatcher()).ConfigureAwait(false);
+    }
+
+    protected override void HandleWorkerConnectionEstablished()
+    {
+        // Add all the systems that you want on your workers.
+        Worker.World.GetOrCreateManager<YourSystem>();
+    }
+
+    protected override void HandleWorkerConnectionFailure()
+    {
+        Debug.Log("Failed to create worker.");
+    }
+
+    public override void Dispose()
+    {
+        // Need to call base.Dispose() to properly clean up.
+        // The ECS world and the connection.
+        base.Dispose();
+    }
+}
+```
+
+<%(/Expandable)%>
+
+## How to modify the connection configuration
+
+When configuring the connection, you have a number of parameters to configure. The GDK provides a small framework for composing these parameters which you can extend to build reusable configurations.
+
+The `SpatialOSConnectionHandlerBuilder` object is central to this framework, it allows you to declare which connection flow to use and what parameters the connection should use. There are two key decisions that you need to make when configuring your connection.
+
+### What connection parameters should I use
+
+The [`WorkerConnector`]({{urlRoot}}/api/core/worker-connector) class has a `CreateConnectionParameters(string workerType)` method which returns reasonable defaults for your `ConnectionParameters`. You can then tweak these to fit your use case:
```

Comment:
maybe? idm either way `...your use case. For example: `

Reply:
Yeah I think I prefer how you've written it